main_test.go

```diff
@@ -226,11 +224,11 @@ func TestWriteDataWithFilters(t *testing.T) {
 	}
 
 	for _, tc := range tcs {
+		tc := tc
```

Comment:
why self assign? lets remove it

Reply:
Because linter was throwing a "loop variable tc captured by func literal" warning, this solved it